models/autoencoders/autoencoder.py

```diff
@@ -67,7 +67,8 @@ def forward(self, input_data):
     
     def loss(self, input_data, fwd_out):
         loss=self._loss_fct(fwd_out.output_data, input_data.view(-1,self._flat_input_size), reduction='sum')
-        return loss
+        return {"loss":loss}
```

Comment:
This is how wandb starts logging metrics, right? Should I add this change to the other models too? Maybe we can automatise this using the inheritance structure we have, i.e. adding that to the `autoencoderbase`.